sunset/code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -160,3 +160,158 @@
+/datum/sprite_accessory/ipc_screens/green
+	name = "Green"
+	icon_state = "green"
+/datum/sprite_accessory/ipc_screens/heart
+	name = "Heart"
+	icon_state = "heart"
+	color_src = 0
+/datum/sprite_accessory/ipc_screens/monoeye
+	name = "Mono-eye"
+	icon_state = "monoeye"
+/datum/sprite_accessory/ipc_screens/nature
+	name = "Nature"
+	icon_state = "nature"
+/datum/sprite_accessory/ipc_screens/orange
+	name = "Orange"
+	icon_state = "orange"
+/datum/sprite_accessory/ipc_screens/pink
+	name = "Pink"
+	icon_state = "pink"
+/datum/sprite_accessory/ipc_screens/purple
+	name = "Purple"
+	icon_state = "purple"
+/datum/sprite_accessory/ipc_screens/rainbow
+	name = "Rainbow"
+	icon_state = "rainbow"
+	color_src = 0
+/datum/sprite_accessory/ipc_screens/red
+	name = "Red"
+	icon_state = "red"
+/datum/sprite_accessory/ipc_screens/redtext
+	name = "Red Text"
+	icon_state = "redtext"
+	color_src = 0
+/datum/sprite_accessory/ipc_screens/rgb
+	name = "RGB"
+	icon_state = "rgb"
+/datum/sprite_accessory/ipc_screens/scroll
+	name = "Scanline"
+	icon_state = "scroll"
+/datum/sprite_accessory/ipc_screens/shower
+	name = "Shower"
+	icon_state = "shower"
+/datum/sprite_accessory/ipc_screens/sinewave
+	name = "Sinewave"
+	icon_state = "sinewave"
+/datum/sprite_accessory/ipc_screens/squarewave
+	name = "Square wave"
+	icon_state = "squarewave"
+/datum/sprite_accessory/ipc_screens/static_screen
+	name = "Static"
+	icon_state = "static"
+/datum/sprite_accessory/ipc_screens/yellow
+	name = "Yellow"
+	icon_state = "yellow"
+/datum/sprite_accessory/ipc_antennas
+	icon = 'sunset/icons/mob/ipc_accessories.dmi'
+	color_src = HAIR
+/datum/sprite_accessory/ipc_antennas/none
+	name = "None"
+	icon_state = "None"
+/datum/sprite_accessory/ipc_antennas/angled
+	name = "Angled"
+	icon_state = "antennae"
+/datum/sprite_accessory/ipc_antennas/antlers
+	name = "Antlers"
+	icon_state = "antlers"
+/datum/sprite_accessory/ipc_antennas/crowned
+	name = "Crowned"
+	icon_state = "crowned"
+/datum/sprite_accessory/ipc_antennas/cyberhead
+	name = "Cyberhead"
+	icon_state = "cyberhead"
+/datum/sprite_accessory/ipc_antennas/droneeyes
+	name = "Drone Eyes"
+	icon_state = "droneeyes"
+/datum/sprite_accessory/ipc_antennas/light
+	name = "Light"
+	icon_state = "light"
+/datum/sprite_accessory/ipc_antennas/sidelights
+	name = "Sidelights"
+	icon_state = "sidelights"
+/datum/sprite_accessory/ipc_antennas/tesla
+	name = "Tesla"
+	icon_state = "tesla"
+/datum/sprite_accessory/ipc_antennas/tv
+	name = "TV Antenna"
+	icon_state = "tvantennae"
+/datum/sprite_accessory/ipc_chassis // Used for changing limb icons, doesn't need to hold the actual icon. That's handled in ipc.dm
+	icon = null
+	icon_state = "who cares fuck you" // In order to pull the chassis correctly, we need AN icon_state(see line 36-39). It doesn't have to be useful, because it isn't used.
+	color_src = 0
+/datum/sprite_accessory/ipc_chassis/mcgreyscale
+	name = "Morpheus Cyberkinetics(Greyscale)"
+	limbs_id = "mcgipc"
+	color_src = MUTCOLORS

```

Comment:
color_src = MUTCOLORS doesn't allow for darker colors (due to unathi), like black or dark grey. This doesn't seem to be much of a balance issue, but is quite annoying for those wanting a non-neon edgy snowflake IPC.

Reply:
It only blocks colors so dark you'd lose detail anyway. This is how we had it on Oracle. I would advise against attempting to reinvent the `MUTCOLORS` wheel